### DISCUSSION TITLE
查询执行 | Databend 内幕大揭秘

### DISCUSSION THREAD

**Discussion by giscus** [PsiACE/databend-internals#2] [Announcements]
# 查询执行 | Databend 内幕大揭秘

查询在数据库系统中的调度与执行方式同样会深刻影响到数据库的性能表现。本文简要梳理了查询流程中与执行相关的内容，希望能够帮助大家更好理解查询引擎的工作原理。

https://psiace.github.io/databend-internals/docs/the-basics/executor-in-query-process/

**Comment by fkuner** [comment]
学习了^-^
